worlds/shorthike/docs/setup_en.md

```diff
@@ -14,18 +13,13 @@
 
 ## Installation
 
-1. Open the [Modding Tools GitHub page](https://github.com/BrandenEK/AShortHike.ModdingTools/), and follow
-the installation instructions. After this step, your `A Short Hike/` folder should have an empty `Modding/` subfolder.
-
-2. After the Modding Tools have been installed, download the 
-[Randomizer](https://github.com/BrandenEK/AShortHike.Randomizer/releases) zip, extract it, and move the contents
-of the `Randomizer/` folder into your `Modding/` folder. After this step, your `Modding/` folder should have
- `data/` and `plugins/` subfolders.
+Open the [Randomizer Repository](https://github.com/BrandenEK/AShortHike.Randomizer) and follow
```

Comment:
I'm a bit confused, this link goes to a repo that says "**Installation** Follow the instructions on the [Mod Installer](https://github.com/BrandenEK/AShortHike.Modding.Installer) page for automatic or manual installation" So you're having the user go to a repo exclusively to go to another repo?

Reply:
This is because the installer is now the one place where detailed installation instructions will be from now on. However, they are generic, and the Randomizer GitHub states which dependencies are also required to download